utils/parseOlderThan.js

```diff
@@ -16,6 +16,11 @@ function parseOlderThan(olderThan) {
         cutoff.setDate(cutoff.getDate() - numberOfDays);
         return cutoff;
     }
+    const numberOfSecsMatch = /^([0-9]+) seconds?$/.exec(olderThan);
```

Comment:
Can you also add a li'l test [here](https://github.com/scality/s3utils/blob/development/1.14/tests/unit/utils/parseOlderThan.js)? :wink: